Description: Fix warning on large files
 On files > 2GB on 32-bit systems, symlinks would print annyoing error
 messages "Value too large for defined data type".
Author: Eduard Bloch <edi@gmx.de>
Bug-Debian: http://bugs.debian.org/167122

---
 symlinks.c |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

--- a/symlinks.c
+++ b/symlinks.c
@@ -1,3 +1,7 @@
+#define _FILE_OFFSET_BITS 64
+#define _LARGEFILE_SOURCE
+#define _LARGEFILE64_SOURCE
+
 #include <unistd.h>
 #ifndef _POSIX_SOURCE
 #define _POSIX_SOURCE
